cmd/healthbot/main.go

```diff
@@ -35,11 +38,24 @@ func main() {
 		if err != nil {
 			log.Fatal().Err(err).Msgf("receipt timeout has invalid format: %s", chainCfg.Probe.ReceiptTimeout)
 		}
+		wallet, err := wallet.NewWallet(chainCfg.Signer.PrivateKey)
+		if err != nil {
+			log.Fatal().Err(err).Msg("unable to create wallet from private key string")
+		}
+
+		chain, err := getChain(chainCfg.Name)
```

Comment:
Getting the chain information to initialize the Go client with from the chain name provided in the config.